GenericJSON/Initialization.swift

```diff
@@ -96,3 +98,21 @@ extension JSON: ExpressibleByStringLiteral {
         self = .string(value)
     }
 }
+
+// MARK: - NSNumber
+
+extension NSNumber {
+    fileprivate var isBool: Bool {
+        let objCType = String(cString: self.objCType)
+        if (self.compare(trueNumber) == .orderedSame && objCType == trueObjCType) || (self.compare(falseNumber) == .orderedSame && objCType == falseObjCType) {
+            return true
+        } else {
+            return false
+        }
+    }
+}
+
+private let trueNumber = NSNumber(value: true)
```

Comment:
Could we please move these symbols to the extension above? Since they are not used from anywhere else, we can make it obvious by limiting their visibility.

Reply:
Since extensions are not allowed to have stored properties I don't think this is possible unfortunately.

Reply:
🤦‍♂ I always forget. They could be static, but that would complicate the access, so let’s just ignore it.